teams/teams-ps/teams/Get-CsTeamsShiftsConnectionConnector.md

```diff
@@ -0,0 +1,55 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: Get-CsTeamsShiftsConnectionConnector
+author: gucsun
+manager: navinth
+online version: https://docs.microsoft.com/powershell/module/teams/get-csteamsshiftsconnectionconnector
+schema: 2.4.0
```

Comment:
This does not refer to the module version but docs version, it should be "2.0.0"